ui/app/AppLayouts/Chat/views/ChatHeaderContentView.qml

```diff
@@ -0,0 +1,344 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+import utils 1.0
+
+import "../panels"
+
+RowLayout {
+    id: root
+    anchors.fill: parent
```

Comment:
Uhm, not really nice

Reply:
why not? Its parent is an Item
```
        Item {
            id: headerContentItem
            Layout.fillWidth: true
            Layout.fillHeight: true
        }
 ```

Reply:
You should really not refer to `parent` at the toplevel in a standalone component

Reply:
oh, true! removed :)